docker_test.go

```diff
@@ -443,18 +523,117 @@ func TestWebhook(t *testing.T) {
 	require.Equal(t, gjson.GetBytes(body, "anonymousId").Str, "anon-id-new")
 	require.Equal(t, gjson.GetBytes(body, "userId").Str, "identified user id")
 	require.Equal(t, gjson.GetBytes(body, "rudderId").Str, "daf823fb-e8d3-413a-8313-d34cd756f968")
-	require.Equal(t, gjson.GetBytes(body, "type").Str, "track")
-
-
-	// TODO: Verify in Live Evets API
+	require.Equal(t, gjson.GetBytes(body, "type").Str, "identify")
+	
 }
 // Verify Event in POSTGRES
 func TestPostgres(t *testing.T) {
 	var myEvent Event
 	require.Eventually(t, func() bool {
-		eventSql:= "select anonymous_id, user_id from example.tracks limit 1"
+		eventSql:= "select anonymous_id, user_id from example.identifies limit 1"
 		db.QueryRow(eventSql).Scan(&myEvent.anonymous_id, &myEvent.user_id)
 		return myEvent.anonymous_id == "anon-id-new"
 	}, time.Minute, 10*time.Millisecond)
 	require.Equal(t, "identified user id", myEvent.user_id)
+	
+	}
+// Verify Event in Redis	
+func TestRedis(t *testing.T) {
+	fmt.Println(address)
```

Comment:
```suggestion
	log.Println("redisHost:", address)
```
Also I would suggest moving this in setup